NEWS.md

```diff
@@ -1,6 +1,565 @@
+* Improve mkldnn fallback. (#12663)
+* Update MKL-DNN dependency (#12953)
+* Update MKLML dependency (#13181)
+* [MXNET-33] Enhance mkldnn pooling to support full convention (#11047)
+
+#### Inference
+* [MXNET-910] Multithreading inference. (#12456)
+* Tweaked the copy in c_predict_api.h (#12600)
+
+#### Other
+* support for upper triangular matrices in linalg (#12904)
+* [MXNET-918] Introduce Random module / Refact code generation (#13038)
+* [MXNET-779]Add DLPack Transformation API (#12047)
+* Draw labels name (#9496)
+* Change the way NDArrayIter handle the last batch (#12285)
+* Revert Change the way NDArrayIter handle the last batch (#12537)
+* Track epoch metric separately (#12182)
+* Set correct update on kvstore flag in dist_device_sync mode (#12786)
+
+### Frontend API updates
+
+#### Gluon
+
+* Update basic_layers.py (#13299)
+* Gluon LSTM Projection and Clipping Support (#13056)
+* Make Gluon download function to be atomic (#12572)
+* [MXNET -1004] Poisson NegativeLog Likelihood loss (#12697)
+* add activation information for mxnet.gluon.nn._Conv (#12354)
+* Gluon DataLoader: avoid recursionlimit error (#12622)
+
+#### Symbol
+* Addressed dumplicate object reference issues (#13214)
+* Throw exception if MXSymbolInferShape fails. (#12733)
+* Infer dtype in SymbolBlock import from input symbol (#12412)
+
+### Language API updates
+#### Java
+* [MXNET-1198] MXNet Java API (#13162)
+
+#### R
+* Refactor R Optimizers to fix memory leak - 11374
+* Add new Vignettes to the R package
+  * Char-level Language modeling - 12670
+  * Multidimensional Time series forecasting - 12664
+* Fix broken Examples and tutorials
+  * Tutorial on neural network introduction - 12117
+  * CGAN example - 12283
+  * Test classification with LSTMs - 12263
+
+#### Scala
+* explain the details for Scala Experimental (#12348)
+* MXNET-873 - Bring Clojure Package Inline with New DataDesc and Layout in Scala Package (#12387)
+* [MXNET-716] Adding Scala Inference Benchmarks (#12721)
+* [MXNET-716][MIRROR #12723] Scala Benchmark Extension pack (#12758)
+* NativeResource Management in Scala (#12647)
+* Ignore generated scala files. (#12928)
+* use ResourceScope in Model/Trainer/FeedForward.scala (#12882)
+* [MXNET-1180] Scala Image API (#12995) 
+* ONNX export: Scalar, Reshape - Set appropriate tensor type (#13067)
+* Port of scala Image API to clojure (#13107) 
+* update log4j version of Scala package (#13131)
+* review require() usages to add meaningful messages. (#12570)
+
+#### Clojure
+* Introduction to Clojure-MXNet video link. (#12754)
+* Improve the Clojure Package README to Make it Easier to Get Started (#12881)
+
+#### Perl
+* [MXNET-1026] [Perl] Sync with recent changes in Python's API (#12739)
+
+#### Julia
+* import Julia binding
+
+### Performance improvements
+* update mshadow for omp acceleration when nvcc is not present  (#12674)
+* [MXNET-860] Avoid implicit double conversions (#12361)
+
+### Bug fixes
```

Comment:
My fix to shape inference is missing: https://github.com/apache/incubator-mxnet/pull/13409

https://github.com/apache/incubator-mxnet/pull/13487